cinch/roles/jenkins_common/defaults/main.yml

```diff
@@ -7,3 +7,5 @@ jenkins_ssh_keyfile: id_rsa
 java_version: 8
 # Needed for certain OpenShift instances
 jenkins_user_uid: 1000090000
+# Set the default (empty) set of authorized keys for jenkins user.
```

Comment:
You might want to add a note that this should be an array of file names on the local path pointing to public keys, so someone doesn't try to add the key values directly into this array or blindly add private keys or some other such nonsense.